### PR TITLE
[zh] Add basic cron syntax in documentation

### PR DESCRIPTION
**Pull request by howieyuen**
sync from https://github.com/kubernetes/website/pull/25078

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -85,6 +85,54 @@ This example CronJob manifest prints the current time and a hello message every
...
+| @daily (or @midnight)			| Run once a day at midnight																	| 0 0 * * * 		|
+| @hourly 									| Run once an hour at the beginning of the hour								| 0 * * * * 		|
+-->
+| 输入                    | 描述                            | 相当于         |
+| -------------           | -------------                  |-------------   |
+| @ 每年                  | 每年 1 月 1 日的午夜运行一次      | 0 0 1 1 *      |
+| @ 每月                  | 每月第一天的午夜运行一次          | 0 0 1 * *      |
+| @ 每周                  | 每周的周日午夜运行一次            | 0 0 * * 0      |
+| @ 每天（或者 @ 午夜）    | 每天午夜运行一次                  | 0 0 * * *      |
+| @ 每小时                | 每小时的开始一次                  | 0 * * * *      |
```
Comment (sftim) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:121]: The first column is literals and should not be translated.

**Review by sftim — CHANGES_REQUESTED** [review]

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/concepts/workloads/controllers/cron-jobs.md:121]: copy

File: content/zh/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -85,6 +85,54 @@ This example CronJob manifest prints the current time and a hello message every
...
+| @daily (or @midnight)			| Run once a day at midnight																	| 0 0 * * * 		|
+| @hourly 									| Run once an hour at the beginning of the hour								| 0 * * * * 		|
+-->
+| 输入                      | 描述                            | 相当于         |
+| -------------             | -------------                  |-------------   |
+| @yearly (or @annually)		| 每年 1 月 1 日的午夜运行一次     | 0 0 1 1 *      |
+| @monthly 									| 每月第一天的午夜运行一次         | 0 0 1 * *      |
+| @weekly 									| 每周的周日午夜运行一次           | 0 0 * * 0      |
+| @daily (or @midnight)			| 每天午夜运行一次                 | 0 0 * * *      |
+| @hourly 									| 每小时的开始一次                 | 0 * * * *      |
```
Comment (sftim) [inline content/zh/docs/concepts/workloads/controllers/cron-jobs.md:121]: BTW you _can_ localize “or” in column 1

**Pull request merged**